examples/ai-billing/components/image-generator.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+import { Button, Card, TextInput } from "@tremor/react";
+import { useState } from "react";
+import { toast } from "sonner";
+import { revalidate } from "../app/revalidate";
+
+export function ImageGenerator({ credits }: { credits: number }) {
+  const [prompt, setPrompt] = useState("");
+  const [loading, setLoading] = useState(false);
+  const [images, setImages] = useState([] as string[]);
```

Comment:
looks less hacky, wdyt?

```suggestion
  const [images, setImages] = useState<string[]>([]);
```